examples/README.md

```diff
@@ -0,0 +1,38 @@
+# Examples
+## Modifying the default HiveMQ K8s Image Non-Root Image
+Security is important [[1](https://snyk.io/blog/10-kubernetes-security-context-settings-you-should-understand)] when running containers in Kubernetes. 
+For this reason we provide the documentation on how to modify the HiveMQ image without increasing its size.
+### Build the image
+
+- Get the latest version from the docker registry https://hub.docker.com/r/hivemq/hivemq4/tags?page=1&name=k8s, for instance `hivemq/hivemq4:k8s-<version>`.
+- Build the custom image using the command:
+```bash
+docker build \
+  --build-arg HIVEMQ_IMAGE=hivemq/hivemq4:k8s-<version> \
+  --build-arg JAVA_IMAGE=openjdk:11-jre-slim \
+  -t <custom-org>:<custom-tag> -f example_nonroot_k8s.dockerfile .
+```
+### Configure the Helm-Chart
+- Configure the image name on the helm-chart `values.yml`
```

Comment:
Override the default docker image name of the HiveMQ operator helm chart, by creating a custom 'values.yml' file and configure the podSecurityContext according to your image.